embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/assertions/SpanAssertions.kt

```diff
@@ -24,38 +23,38 @@ import org.junit.Assert.assertTrue
  * Assert the [EmbraceSpanData] is as expected
  */
 internal fun assertEmbraceSpanData(
-    span: EmbraceSpanData?,
+    span: Span?,
```

Comment:
After this, can we rename this to `SpanData` to be more inline with the OTel interface that this extends. We should probably look at all the naming of these internal classes once the migration is done - and also the wrapper stuff I'm doing to implement the OTel API too

Reply:
Sounds good. I will probably wait to rename until the integration branch gets merged around to avoid making merge conflicts even worse than they will be